core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/components/dot-edit-page-info/dot-edit-page-info.component.ts

```diff
@@ -17,6 +17,7 @@ export class DotEditPageInfoComponent {
     @Input() url: string;
     innerApiLink: string;
     previewUrl: string;
+    baseUrl: string = window.location.href.includes('edit-page') ? window.location.origin : '';
```

Comment:
Please use better the injection token of angular to get the defaultWindows.

```
constructor(@Inject(DOCUMENT) private document: Document) {
        console.log(document.defaultView.location.origin);
        console.log(document.defaultView.location.href);
    }`

Reply:
Sure!

Reply:
Done
<img width="783" alt="image" src="https://user-images.githubusercontent.com/127987858/236525941-e823cd66-c57b-4dfa-b298-c41d1d08cb8e.png">